kANe/v3.md

```diff
@@ -17510,6 +17608,8 @@ a yajaman who had employed another priest to perform reli gious ceremonies the a
 
 How customs and practices once prevalent may come to be discontinued or abrogated is well illustrated in the section on Kalivarjyas. Soine of those that have a legal bearing have been already discussed above. 
 
```

Comment:
Lets have
```
## 36
```
here, so that Appendix etc. don't appear to be subsections of chapter 35. If v2 neeeds similar correction, please include that as well.

Reply:
okay, sir.